shelley/design-spec/delegation_design_spec.tex

```diff
@@ -2523,68 +2527,77 @@ \subsection{Overview of Incentives}
 desirable state. If possible, there should never be a conflict of
 interest between maximizing rewards and ``doing the right thing''.
 
-Rewards will be paid for each epoch and will be drawn from two sources,
+Rewards will be paid for each epoch and will be drawn from the following
+sources:
 
 \begin{itemize}
 \item
-  monetary expansion and
+  monetary expansion
+\item
+  transaction fees
 \item
-  transaction fees.
+  decayed deposits.
 \end{itemize}
 
-Rewards for one epoch will roughly be split proportional to stake.
-However, there will be several refinements to this general principle:
+All rewards will be collected in a (virtual) pot, and then shared amongst stake
+pools depending on their contribution to the operation of the system. The main
+factor will be the relative stake that a pool controls. However, there will be
+several refinements to this general principle:
 
 \begin{itemize}
 \item
   Rewards for a stake pool will be capped when the pool gets too large
   (otherwise, the system would converge towards a state with all stake
   being delegated to one giant stake pool).
 \item
-  Rewards will decrease if a pool leader does not create the blocks they
+  Rewards will decrease if a pool operator does not create the blocks they
   are supposed to create.
 \item
-  Pool leaders will be compensated for their trouble and risk by
-\item
-  reimbursing their costs and
-\item
-  giving them a \emph{margin} before distributing pool rewards
-  proportionally amongst pool leader and pool members. (Pool leaders
-  publicly declare their margin, which they can freely choose.)
+  Pool operators will be compensated for their trouble and risk by
+  \begin{itemize}
+  \item
+    reimbursing their costs and
+  \item
+    giving them a \emph{margin} before distributing pool rewards proportionally
+    amongst pool operator and pool members. (Pool operators publicly declare their
+    cost and margin, which they can freely choose.)
+  \end{itemize}
 \item
-  Pool rewards will slightly increase with the stake of their leader.
-  There is no minimal stake required to create a pool -- anybody can do
-  this. However, pools led by leaders with high stake will get higher
-  rewards. (This will discourage pool leaders from splitting their stake
-  to operate several pools. It will also help preventing Sybil attacks,
-  where an attacker with low stake tries to gain control over a majority
-  of stake by creating a lot of pools with low costs.)
+  Pool rewards will slightly increase with the stake their operator contributes to
+  the pool. There is no minimal stake required to create a pool -- anybody can
+  do this. However, pools led by operators with high stake will get higher
+  rewards. (This will discourage pool operators from splitting their stake to
+  operate several pools. It will also help preventing Sybil attacks, where an
+  attacker with low stake tries to gain control over a majority of stake by
+  creating a lot of pools with low costs.)
```

Comment:
Shouldn't it say "owner" whenever pledge/stake is mentioned? The rest of the spec seems to emphasising how pool operator is different from people funding the pool.

Reply:
Right, thanks!

Reply:
Added a commit to consistently use "owner's stake".

Reply:
Thank you!